tf2_kdl_py/package.xml

```diff
@@ -0,0 +1,24 @@
+<package format="2">
+  <name>tf2_kdl_py</name>
+  <version>0.36.1</version>
+  <description>
+    PyKDL binding for tf2
+  </description>
+
+  <maintainer email="alejandro@openrobotics.org">Alejandro Hernandez Cordero</maintainer>
+  <maintainer email="clalancette@openrobotics.org">Chris Lalancette</maintainer>
+
+  <license>Apache-2.0</license>
+
+  <depend>builtin_interfaces</depend>
+  <depend>geometry_msgs</depend>
+  <depend>orocos_kdl_vendor</depend>
+  <depend>tf2_ros_py</depend>
+
+  <exec_depend>tf2_ros_py</exec_depend>
```

Comment:
The dependencies aren't quite right.

First, all dependencies in a python package should be either `exec_depend` or `test_depend` (unless we are compiling something in the package, which isn't the case here).

Second, this package should not directly depend on `orocos_kdl_vendor`.  Instead, it really depends on `python_orocos_kdl_vendor`.

Third, it looks like the `builtin_interfaces` dependency is only for tests, so we can make that a `test_depend`.

Fourth,  we need an `test_depend` on `rclpy`.

And finally, we should have a `test_depend` on `python3-pytest` here, as we technically require that for tests to run.

The result of all of that is that I think this should look like the following:

```suggestion
  <exec_depend>geometry_msgs</exec_depend>
  <exec_depend>python_orocos_kdl_vendor</exec_depend>
  <exec_depend>tf2_ros_py</exec_depend>
  
  <test_depend>builtin_interfaces</test_depend>
  <test_depend>rclpy</test_depend>
  <test_depend>python3-pytest</depend>
```